Tasks/Xcode/xcode.ts

```diff
@@ -231,11 +231,17 @@ async function run() {
             xcb.pipeExecOutputToTool(xcPrettyTool);
         }
 
-        //--- Xcode Build ---
-        await xcb.exec();
+        let xcodeTaskErrors;
+        try {
+            //--- Xcode Build ---
+            await xcb.exec();
+        } catch (err) {
+            xcodeTaskErrors = err;
+            tl.debug('Xcode task failed: ' + err);
```

Comment:
Not sure if it matters, but err doesn't have to be a string. I am not sure how the concatenation would work if err is an Error.

Reply:
Seems to work fine, I see this in the log: ##[debug]Xcode task failed: Error: /usr/bin/xcodebuild failed with return code: 65